packages/jingoo/jingoo.1.3.1/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
```

Comment:
I am a bit confused, `jingoo` version 1.3.1 is already released: https://github.com/ocaml/opam-repository/blob/master/packages/jingoo/jingoo.1.3.1/opam

Reply:
Hmm.. I've just added `jingoo.1.3.2` directory to `opam-repository/packages/jingoo` and push it, then pushed 'New pull request' button from Github UI as always.

Offcource I didn't intend to re-release jingoo1.3.1 by this pull request.

Reply:
Maybe the history got a bit entangled. Can you try to rebase on the master branch of the opam-repository?

Reply:
I've tried to rebase my local opam-repository, and fix some conflict and pushed 'new pull request' button, but same kind of PR is generated.

What I've tried to is ...

```bash
git fetch upstream
git merge upstream/master
git rebase upstream/master
... fix some conflict of (jingoo1.3.0/opam, jingoo1.3.1/opam)
git pull
git push origin master
```

Is it safe if I remove `tategakibunko/opam-repository` and re-fork `ocaml/opam-repository` and re-release jingoo1.3.2?

Reply:
It would be safe, otherwise you could `git reset —hard` your branch with respect to the master branch of the upstream opam repo and then cherry pick the relevant change: `git cherry-pick 8c05efa`. 

I have a copy of the commit on my laptop now. If you want we can also close this PR and I can submit one with your commit in it.